package.json

```diff
@@ -22,17 +22,17 @@
     "etch": "0.12.6",
     "fs-plus": "^3.0.0",
     "glob": "^7.1.1",
-    "js-yaml": "^3.8.4",
+    "js-yaml": "^3.11.0",
```

Comment:
In general I'd prefer it if we don't mix in random package upgrades in PRs which are supposed to resolve something entirely unrelated, but let's keep it like this for this PR.